docs/user-guide/src/tour/final.md

```diff
@@ -1,3 +1,10 @@
 # Final List Code
 
-**TODO**
+## Full Code
+
+```rust,noplaypen
+// Expand to see the full code
+{{#rustdoc_include ../../../../prusti-tests/tests/verify/pass/user-guide/peek_mut_pledges.rs:nothing}}
+```
+
+The next 2 chapters will explain loop invariants and counterexamples, but we will not use the linked list from our previous chapters.
```

Comment:
I would put this sentence at the beginning of the next chapter rather than here. It is easy to adjust the chapters and forget to update this.